rental.rb

```diff
@@ -5,12 +5,13 @@
 class Rental
   attr_accessor :date, :book, :person
 
-  def initialize(date, book, person)
+  def initialize(date, person, book)
     @date = date
-    @book = book
-    @person = person
 
+    @book = book
     book.rentals << self
+
+    @person = person
     person.rentals << self
   end
 end
```

Comment:
- [ ] I am currently unable to create a rental. Check line 57 in app.rb 
![image](https://user-images.githubusercontent.com/81932478/180613266-7d2bf4e0-a37e-4a6c-8c1b-fac1b0459417.png)

Reply:
Hi @Munsa1 , thanks for the feedback.
Let me fix the issue. 🙏🏾